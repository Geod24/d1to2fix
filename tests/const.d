const x = 42;

void main (in char[] args)
{
    const y = 43;
}

struct S
{
    // test that const is converted to enum within struct
    const z = 44;
}
