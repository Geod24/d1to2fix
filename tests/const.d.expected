static immutable x = 42;

void main (in char[] args)
{
    static immutable y = 43;
}

struct S
{
    // test that const is converted to enum within struct
    enum z = 44;
}
